### PR TITLE
Release v.4.0.4

### PR DESCRIPTION
**Pull request by mek-x**

### CODE REVIEW

**Comment by pietrushnic** [comment]
@mek-x something is wrong with your email. Did you added your 3mdeb email to GitHub ? GitHub should recognize your profile.

**Comment by mek-x** [comment]
@pietrushnic now it should be ok

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- nct5104d driver backport
+- check if `git` exists before calling it
+- missing `cbfs_find_string` in `libpayload`
+- executable bit to `xcompile`
+
+### Changed
+- reduced log level: display mainboard, DRAM and ECC info only
+- improved SD card performance
+- forced to use SD in 2.0 mode
+- git repository in `Makefile`
```
Comment (pietrushnic) [inline CHANGELOG.md]: @mek-x this file lack necessary section required by spec. Please take a look at [this](https://raw.githubusercontent.com/pcengines/sortbootorder/coreboot-4.0.x/CHANGELOG.md)

Comment (mek-x) [inline CHANGELOG.md]: added

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+[Unreleased]: https://github.com/pcengines/coreboot/compare/v4.0.4...coreboot-4.0.x
+[v4.0.4]: https://github.com/pcengines/coreboot/compare/v4.0.3...v4.0.4
+[v4.0.3]: https://github.com/pcengines/coreboot/compare/v4.0.2...v4.0.3
+[v4.0.2]: https://github.com/pcengines/coreboot/compare/v4.0.1.1...v4.0.2
+[v4.0.1.1]: https://github.com/pcengines/coreboot/compare/v4.0.1....v4.0.1.1
```
Comment (pietrushnic) [inline CHANGELOG.md]: @mek-x you have to push tags to make it work. For some reason some tags are missing.

**Comment by pietrushnic** [comment]
@mek-x we have some conflicts in config file. Please resolve those.

**Comment by mek-x** [comment]
done

**Pull request merged**